pkg/wireguard/agent/agent.go

```diff
@@ -242,10 +196,8 @@ func (a *Agent) UpdatePeer(nodeName, pubKeyHex string,
 		logfields.PubKey:    pubKeyHex,
 		logfields.NodeIPv4:  nodeIPv4,
 		logfields.PodCIDRv4: podCIDRv4,
-		logfields.WgIPv4:    wgIPv4,
```

Comment:
You also want to remove the constant from the `logfields` package.

Reply:
You are right again!